### PR TITLE
Refs #31014 -- Added FromWKB and FromWKT GIS database functions.

### PR DESCRIPTION
**Pull request by felixxm**
ticket-31014

### CODE REVIEW

**Comment by felixxm** [comment]
buildbot, test on oracle.

**Review by claudep — COMMENTED** [review]
Why did you comment on the ticket that SRID support was not needed? I'm not sure about that. However, SRID support could be added later in a following patch.

File: docs/releases/4.2.txt [hunk 1]
```diff
@@ -161,6 +161,11 @@ Minor features
   :class:`IsEmpty() <django.contrib.gis.db.models.functions.IsEmpty>`
   expression allow filtering empty geometries on PostGIS.
 
+* The new :class:`FromWKB() <django.contrib.gis.db.models.functions.FromWKB>`
+  and :class:`FromWKT() <django.contrib.gis.db.models.functions.FromWKT>`
+  functions allow creating geometries from Well-known binary (WKB) and
+  Well-known text (WKT) representation.
```
[the remark this answers is not in the document]
Comment (felixxm) [reply docs/releases/4.2.txt]: Good catch :dart: Updated.

**Comment by felixxm** [comment]
> Why did you comment on the ticket that SRID support was not needed? I'm not sure about that. However, SRID support could be added later in a following patch.

`srid` is not supported on Oracle, so I didn't want to complicate Sergey's implementation.

**Comment by claudep** [comment]
We can use that patch as is, but sooner or later we'll receive the request to support the SRID argument. Missing Oracle support is not decisive IMHO.

**Comment by felixxm** [comment]
> We can use that patch as is, but sooner or later we'll receive the request to support the SRID argument. Missing Oracle support is not decisive IMHO.

I'm just pretty busy this week :tired_face: :wink: and my knowledge of GIS functions is limited :disappointed:

**Comment by felixxm** [comment]
I can change this to `Refs` and leave the ticket open.

**Comment by claudep** [comment]
Sure, this would be a good option.

**Review by carltongibson — APPROVED** [review]
Looks good. Thanks.

**Comment by felixxm** [comment]
Thanks both :gift: :gift:

**Pull request merged**